src/viur/core/modules/script.py

```diff
@@ -162,3 +163,40 @@ def update_path(self, skel):
             key = parent_skel["parententry"]
 
         skel["path"] = "/".join(path)
+
+    @exposed
+    def get_importable(self):
+
+        # get importable key
+        qry_importable = self.viewSkel("node").all().filter("parententry", self.rootnodeSkel(ensure=True)["key"])
+        if not (qry_importable := self.listFilter(qry_importable)):
+            raise errors.Unauthorized()
+        importable_key = None
+        for data in qry_importable.iter():
+            if data["name"] == "importable":
+                importable_key = data.key
+                break
```

Comment:
Why do you not filter this in the db?
```suggestion
        qry_importable = self.viewSkel("node").all().filter("parententry", self.rootnodeSkel(ensure=True)["key"]).filter("name =", "importable")
        if not (qry_importable := self.listFilter(qry_importable)):
            raise errors.Unauthorized()
        importable_key = (entity := qry_importable.getEntry()) and entity["key"]
```